backend/api/Areas/Leases/Models/Lease/AddressModel.cs

```diff
@@ -77,6 +77,11 @@ public class AddressModel
         /// </summary>
         public string Province { get; set; }
 
+        /// <summary>
+        /// get/set - The code name of the province.
+        /// </summary>
+        public string ProvinceCode { get; set; }
+
```

Comment:
I think other operations implemented by Manuel he has used nested objects vs flattening individual properties into the model. I'm fine with either but would like that we agree on a consistent way to model our API

Reply:
This one is already flattened, so not planning on fixing the overall model in this PR.

Reply:
but for new models we should not be flattening.

Reply:
Fair enough. This might also be a symptom of reusing the same model for read and create